### PR TITLE
Update onto-docker.md

### PR DESCRIPTION
**Pull request by JannisGrundmann**
Deployment guide updated

### CODE REVIEW

File: docs/pages/services/onto-docker.md [hunk 1]
```diff
@@ -34,27 +33,27 @@ git clone https://git.material-digital.de/apps/ontodocker.git
...
 ```
 
-### 2. Copy compose template
+### 2. Connect to SSO Identity Provider (IDP)
 
-```bash
-# copy (or link) the compose file template
-cp docker-compose-prod.yml docker-compose.yml
-```
+If you alreday have a valid client_secrets.json already, you must copy it to ```./data/oidc/```
```
Comment (schaarj) [inline docs/pages/services/onto-docker.md:38]:
```suggestion
> **Note:** If you have a valid client_secrets.json already, place it in `./data/oidc/`
{: .warning }
```

File: docs/pages/services/onto-docker.md [hunk 2]
```diff
@@ -34,27 +33,27 @@ git clone https://git.material-digital.de/apps/ontodocker.git
...
+
+Otherwise in order to connect OntoDocker to the IDP you might need an initial access token (IAT) or generate one, if you want to connect it to your local Instance. (see "Initial Access Token" section of the [Keycloak manual](https://www.keycloak.org/docs/latest/securing_apps/#_initial_access_token))
```
Comment (schaarj) [inline docs/pages/services/onto-docker.md:40]:
```suggestion
In order to connect OntoDocker to the IDP you might need an initial access token (IAT), which you will receive from the maintainer of the IDP,  or generate one, if you want to connect it to your local Instance. (see "Initial Access Token" section of the [Keycloak manual](https://www.keycloak.org/docs/latest/securing_apps/#_initial_access_token))
```

we should probably add a section about IAT's in our deployment guide but for now Keycloak manual is also fine

File: docs/pages/services/onto-docker.md [hunk 3]
```diff
@@ -34,27 +33,27 @@ git clone https://git.material-digital.de/apps/ontodocker.git
...
 
-### 3. Connect to SSO Identity Provider (IDP)
-In order to connect OntoDocker to the IDP you might need an initial access token (IAT) or generate one, if you want to connect it to your local Instance. (see "Initial Access Token" section of the [Keycloak manual](https://www.keycloak.org/docs/latest/securing_apps/#_initial_access_token))
+Save the Initial Access Token as ```ia.jwt```
 
+Replace KEYCLOAK_URL, REALM_NAME in ```provider_info.json``` accordingly to the IDPs properties
+
+Set the APPLICATION_URL to the url where your instance is supposed to be accessible at
+
+Copy both ```ia.jwt``` and the customized ```provider_info.json``` files to ```./data/oidc/```
```
Comment (schaarj) [inline docs/pages/services/onto-docker.md:48]: Add some point we should probably try to have all configuration parameters in a single config.json and use this to automatically populate the custom config files

**Review by schaarj — CHANGES_REQUESTED** [review]
on a general note, are the three backticks on purpose instead of just single backticks? not sure if they'll render the same way or will be handled as separate code block

File: docs/pages/services/onto-docker.md (hunk 1 above)
Comment (JannisGrundmann) [reply docs/pages/services/onto-docker.md:38]: The triple backticks are a mistake. Single backticks are the right way, I agree.

File: docs/pages/services/onto-docker.md (hunk 3 above)
Comment (JannisGrundmann) [reply docs/pages/services/onto-docker.md:48]: Good idea

**Review by schaarj — APPROVED** [review]

**Pull request merged**